code-challenges.test.js

```diff
@@ -10,58 +10,211 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+
+
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
+// const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
+// const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("fibonacci", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6
+    const expectedOutput1 = [1, 1, 2, 3, 5, 8]
+    const fibLength2 = 10
+    const expectedOutput2 = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(fibonacci(fibLength1)).toEqual(expectedOutput1)
+    expect(fibonacci(fibLength2)).toEqual(expectedOutput2)
+  })
+})
+
+//ReferenceError: fibonacci is not defined
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
 // b) Create the function that makes the test pass.
 
+// Input: number
+// Output: array of numbers
+
+// Create a function that takes in a number as a parameter
+// Create if/then statement.
+// In the else statement, create a for loop
+// Return Array
+// I honestly have no idea how to do this. I have been trying the logic for two hours. I still don't get it. This is probably simpler than what I am thinking it is.
+
+
+// const fibLength1 = 6
+// const fibLength2 = 10
+
+const fibonacci = (number) => {
+  let newArray = [0, 1]
+    if (number <= 1) {
+      return "pick a number greater 1"
+  } else {
+      for (let i = 1; i < number; i++) {
+        newArray.push(newArray[i] + newArray[i - 1])
+      }
+  } return newArray
+}
+
+// Fail:
+//expect(received).toEqual(expected) // deep equality
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
```

Comment:
This is good work! If you record the outcome you are getting then I can help you understand what's happening. 
ex. [0,1,1,2...]